spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/MklInt8Convertible.scala

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.intel.analytics.bigdl.nn.mkldnn
+
+import scala.collection.mutable.ArrayBuffer
+
+/**
+* Trait which provides MKL-DNN functionality to convert FP32 model to INT8 model
+*/
+trait MklInt8Convertible {
+  // input dimension mask
+  protected var inDimMask: Int = 0
```

Comment:
By BigDL conversion, I prefer to use `inputDimMask`, similar below for `outputDimMask`